src/main/resources/templates/java/test/test/${packageNameFolder}/ClassTest.java

```diff
@@ -26,7 +26,13 @@
 @RunWith(Parameterized.class)
 public class ClassTest extends StructuralTest {
```

Comment:
Why are those changes part of your PR? Wasn't this based on this commit that is already merged in develop?
https://github.com/ls1intum/ArTEMiS/commit/0786d2bc5b6f6ad4cafc71f5fd0b7eb472658831#diff-9465e853facabba0aa102a57fe267a03

Reply:
not quite sure how they got into there.. i will try to remove them from the changes

Reply:
well ClassTest.java has the exact same content like the version of it in develop not sure why there are changes !?